puppet/modules/quickstack/manifests/pacemaker/nosql.pp

```diff
@@ -92,7 +92,7 @@
     Exec['all-nosql-nodes-are-up'] ->
 
     quickstack::pacemaker::resource::service {'mongod':
-      options        => 'start timeout=10s',
+      options        => 'op start timeout=10s',
```

Comment:
Both variants (using 'pcs resource create [...] op start timeout=10s op monitor start-delay=10s' vs  'pcs resource create [...] op start timeout=10s monitor start-delay=10s') will result in the same configuration, but fair enough, if removing the second op simplifies things then what you propose should be fine from the pacemaker side.